testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/assertions/ProcessInstanceAssertj.java

```diff
@@ -59,6 +66,26 @@ public ProcessInstanceAssert isTerminated() {
     return this;
   }
 
+  @Override
+  public ProcessInstanceAssert hasActiveElements(final String... elementNames) {
+    hasElementsInState(elementNames, FlowNodeInstanceState.ACTIVE, Objects::nonNull);
+    return this;
+  }
+
+  @Override
+  public ProcessInstanceAssert hasCompletedElements(final String... elementNames) {
```

Comment:
💭 It'd be great in the future to have assertions that also verify the order in which the elements reached the state.

Reply:
Yes!